### PR TITLE
Gdxdsd 6863 move local to openshift

### PR DESCRIPTION
**Pull request by veenupunyani**

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -5,10 +5,27 @@ Instructions for configuring and starting the URL-S frontend app, Vue.js, and it
...
+
+### Openshift dev Setup 
+
+To host the app in dev project of Openshift, follow this process:
+
+**`Step 1`**. Setup the app using "Import from Git" strategy and pull the https://github.com/bcgov/GDX-Analytics-URL-Shortener/tree/gdxdsd-6863-move-local-to-openshift/frontend/url-shortener code.
+
+A docker file at frontend/url-shortener/Dockerfile is provided as part of this folder which is enough to provide insutructions to openshift and to start a http server to serve the static files. This should create a pod and route in dev project where frontend will accessible.
+ 
+If you would like to know more about we did the openshift setup like route setting, secrets, services etc, then please contact us.
```
Comment (cnburnett) [inline frontend/README.md:31]: Suggest that we use OpenShift not openshift

File: frontend/README.md [hunk 2]
```diff
@@ -5,10 +5,27 @@ Instructions for configuring and starting the URL-S frontend app, Vue.js, and it
 Before doing the the frontend install and start, the Backend application must be installed and started. Follow instructions in the README in the Backend folder.
 
 ## Local Setup
+
```
Comment (cnburnett) [inline frontend/README.md:8]: Please change this to **Step 1**

File: backend/db.js [hunk 3]
```diff
@@ -3,28 +3,31 @@ import dotenv from 'dotenv';
...
 
 // Log the current environment mode
-console.log('NODE_ENV:', process.env.NODE_ENV);
+//console.log('NODE_ENV:', process.env.NODE_ENV);
 
 // Determine if the environment is local or not
 const isLocal = process.env.NODE_ENV === 'development';
-const mongodbHost = isLocal ? 'localhost' : (process.env.MONGODB_HOST || 'mongodb.c6d33e-tools.svc.cluster.local');
-const mongodbDatabase = 'mongodb';  // Your database name
+const mongodbHost = isLocal ? 'localhost' : (process.env.MONGODB_HOST || 'mongodb-dev');
```
Comment (cnburnett) [inline backend/db.js:10]: Please provide a 
// Explanation of what the IF statement is doing

File: frontend/README.md [hunk 4]
```diff
@@ -5,10 +5,27 @@ Instructions for configuring and starting the URL-S frontend app, Vue.js, and it
...
   
 **`Step 2`**. From the /url-shortener folder, run `npm install` in the terminal to install required dependencies.
  
 **`Step 3`**. From the /url-shortener folder, run `npm run dev` in the terminal to start the app.
 
-**`Step 4`**. Open a browser and visit the app dev site at http://localhost:5173
\ No newline at end of file
+**`Step 4`**. Open a browser and visit the app dev site at http://localhost:5173
+
+### Openshift dev Setup
```
Comment (cnburnett) [inline frontend/README.md:23]: OpenShift dev pod Setup

File: frontend/README.md [hunk 5]
```diff
@@ -5,10 +5,27 @@ Instructions for configuring and starting the URL-S frontend app, Vue.js, and it
...
 
-**`Step 4`**. Open a browser and visit the app dev site at http://localhost:5173
\ No newline at end of file
+**`Step 4`**. Open a browser and visit the app dev site at http://localhost:5173
+
+### Openshift dev Setup 
+
+To host the app in dev project of Openshift, follow this process:
+
+**`Step 1`**. Setup the app using "Import from Git" strategy and pull the https://github.com/bcgov/GDX-Analytics-URL-Shortener/tree/gdxdsd-6863-move-local-to-openshift/frontend/url-shortener code.
```
Comment (cnburnett) [inline frontend/README.md:27]: Remove **Step 1**

**Review by cnburnett — APPROVED** [review]
READMEs looks good with minor text edits. 
Code works as expected.

**Comment by veenupunyani** [comment]
i have updated the PR with suggested changes

**Pull request merged**